tests/test_dem.py

```diff
@@ -5,8 +5,13 @@
 import pytest
 import warnings
 import pyproj
+import inspect
+import xdem
+import numpy as np
 from xdem.dem import DEM
 
+path_module = os.path.dirname(os.path.dirname(os.path.abspath(inspect.getsourcefile(xdem))))
+
 #TODO: move datasets to a "dataset" folder like in geoutils
 EXAMPLE_PATHS = {
     "dem1": "examples/Longyearbyen/data/DEM_2009_ref.tif",
```

Comment:
Could this be adapted to use the new path system?
```python
xdem.examples.download_longyearbyen_data(overwrite=False)

xdem.exampled.FILEPATHS["longyearbyen_ref_dem"]  # This is the same as EXAMPLE_PATHS["dem1"] before.
```

Reply:
> A workaround for the time being could be:
> 
> ```python
> import json
> import subprocess
> 
> import pyproj
> 
> 
> def get_crs(filepath: str) -> pyproj.CRS:
>     """Get the CRS of a raster with the given filepath."""
>     info = subprocess.run(
>         ["gdalinfo", "-json", filepath],
>         stdout=subprocess.PIPE,
>         check=True,
>         encoding="utf-8"
>     ).stdout
> 
>     wkt_string = json.loads(info)["coordinateSystem"]["wkt"]
> 
>     return pyproj.CRS.from_wkt(wkt_string)
> ```
> 
> Does this work for you, @rhugonnet ? It works for me, but I have an updated proj installation so maybe it won't work for you still.

This worked! I've implemented as temporary fix.
Didn't expect PROJ/pyproj to not support some projections at such advanced versions (7.1)... will open an issue. Maybe we can simply force proj>7.2 in the requirements at a later stage.

Reply:
changed all paths from test_dem.py